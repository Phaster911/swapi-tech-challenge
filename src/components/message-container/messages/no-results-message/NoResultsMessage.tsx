import StyledMessageSection from "../../../shared/styled-message-wrapper/StyledMessageWrapper";

export default function NoResultsMessage() {
  return (
    <StyledMessageSection
      aria-labelledby="no-results-title"
      role="status"
      aria-live="assertive"
      data-testid="no-results-message-testid"
    >
      <h2 id="no-results-title">No Results Found</h2>
      <p>
        We're sorry, but no results matched your search criteria. Please try
        again with different keywords.
      </p>
    </StyledMessageSection>
  );
}
