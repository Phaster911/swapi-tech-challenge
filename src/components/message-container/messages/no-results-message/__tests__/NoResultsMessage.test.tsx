import { render } from "@testing-library/react";
import NoResultsMessage from "../NoResultsMessage";

describe("<NoResultsMessage />", () => {
  it("renders the component", () => {
    const { container } = render(<NoResultsMessage />);
    expect(container).toMatchSnapshot();
  });
});
