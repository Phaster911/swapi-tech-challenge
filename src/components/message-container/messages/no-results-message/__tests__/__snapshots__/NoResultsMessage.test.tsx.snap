// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<NoResultsMessage /> > renders the component 1`] = `
.c0 {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  gap: 1rem;
  text-align: center;
}

<div>
  <div
    aria-labelledby="no-results-title"
    aria-live="assertive"
    class="c0"
    data-testid="no-results-message-testid"
    role="status"
  >
    <h2
      id="no-results-title"
    >
      No Results Found
    </h2>
    <p>
      We're sorry, but no results matched your search criteria. Please try again with different keywords.
    </p>
  </div>
</div>
`;
